packages/clarity-js/src/layout/dom.ts

```diff
@@ -140,7 +134,7 @@ export function update(node: Node, parent: Node, data: NodeInfo, source: Source)
                 let childIndex = previousId === null ? values[parentId].children.length : values[parentId].children.indexOf(previousId) + 1;
                 values[parentId].children.splice(childIndex, 0, id);
```

Comment:
will this work if an element is inserted as a first child? - it seems like we will consider it inserted as last child?